### ISSUE TITLE
Misleading message for `baker remove`

### ISSUE THREAD

**Issue by concordium-cl** [Concordium/concordium-client#121]
**Bug Description**
When removing a baker in the concordium-client, there are the following misleading messages, resulting from the fact that deregistering as baker is now the same as decreasing the stake to 0 and thereby below the baker threshold.

```
Warning: The staked amount (0.000000 CCD) is lower than the minimum baker stake threshold (1000.000000 CCD).
This transaction will most likely be rejected by the chain, do you wish to send it anyway [yN]: y
y
Warning: The new staked value appears to be lower than the amount currently staked on chain by this baker.
```

**Expected Result**
When removing a baker/decreasing stake to 0, different warnings should be shown as when just decreasing stake to below baker thresold.

**Actual Result**
Misleading warning that transaction will most likely be rejected.

**Versions**
- concordium-client_4.0.0-delegation-test-1
- Windows

**Comment by concordium-cl** [comment]
<img width="860" alt="remove baker 2" src="https://user-images.githubusercontent.com/69570978/160405721-e865e81d-65c1-45cc-b75b-c7a697e2bde1.PNG">

**Comment by concordium-cl** [comment]
Similar issue for` delegator remove `.

**Comment by concordium-cl** [comment]
<img width="822" alt="delegating2" src="https://user-images.githubusercontent.com/69570978/160801808-5ca7dc63-309a-4d99-beba-eb94bc4766aa.PNG">

**Comment by abizjak** [comment]
Addressed in 4.0.1.
